server/db/migrate/0003_password_tokens_postgres.down.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS access_tokens;
```

Comment:
imho this is risky, if something happens and we need to rollback, this will loose data...

I usually leave the rollback empty. When talking db rollback, its usually not obvious what the right thing to do is

Reply:
good point!